pkg/apis/kops/componentconfig.go

```diff
@@ -1035,6 +1035,21 @@ type ClusterAutoscalerConfig struct {
 	// CustomPriorityExpanderConfig overides the priority-expander ConfigMap with the provided configuration. Any InstanceGroup configuration will be ignored if this is set.
 	// This could be useful in order to use regex on priorities configuration
 	CustomPriorityExpanderConfig map[string][]string `json:"customPriorityExpanderConfig,omitempty"`
+	// ClusterAutoscalerServiceMonitorConfig specifies the configuration for an optional service monitor
```

Comment:
Nit: the comment should be `serviceMonitor specifies...` because that's what makes the most sense in the CRD (and thus in any docs etc).

(There's a debate about whether it should be `ServiceMonitor` or `serviceMonitor`, one is more correct for the go comment and one is more correct for the CRD comment, I guess I'd favor users over developers, so I prefer `serviceMonitor` but I suspect I've made both choices personally in the past, probably depending on what linters I was using at the time!)